<div class="partial-page">

    <view-title>Scoring</view-title>

    <div class="cttv-content-header">
        <div class="container">
            <h3>Scoring target-disease associations</h3>
            <p></p>
        </div>
    </div>

    <div class="container">

        <div class="cttv-docs-section">
            <p>The Open Targets Platform allows you to prioritise targets based on the strength of their association with a disease. We assess the key factors that relate to the confidence in the target-disease association. We provide association scores to help you answering these questions:</p>
            <ul>
                <li>Which targets have the most evidence for association with a disease?</li>
                <li>What is the relative weight of the evidence for different targets associated with a disease?</li>
            </ul>
            <p>The association score is a numerical value varying from 0 to 1, which indicates the strength of the
            association between a target and a disease. A score of 1 refers to the strongest association, whereas a
            score of 0 corresponds to no evidence supporting an association. In our Platform, we represent the
            different scores with varying shades of blue: the darker the blue, the stronger the association.</p>

        </div>


        <div class="cttv-docs-section">
            <h4 id="computing-the-association-score">Computing the Association Score</h4>

            <p>We start by generating a score for each evidence from different data sources (e.g. GWAS catalog, European Variation Archive) within a data type (e.g. Genetic associations). We define the evidence score as:</p>
            <p>s = F * S * C</p>
            <p>where</p>

            <p>s = score</p>
            <p>F = frequency, the relative occurrence of a target-disease evidence</p>
            <p>S = severity, the magnitude or strength of the effect described by the evidence</p>
            <p>C = confidence, overall confidence for the observation that generates the target-disease evidence</p>

            <p>The evidence score summarises the strength of the evidence and depends on factors that affect its relative strength. These factors are specific to the different data sources in the Platform:</p>

            <table class="table">
                <thead>
                    <tr>
                        <th>Data type</th>
                        <th>Data sources and factors that affect the relative strength of the evidence scores</th>
                    </tr>
                </thead>

                <tbody>
                    <tr>
                        <td>Genetic associations</td>
                        <td>GWAS and PheWASCatalog (functional consequence score of <a target=_blank href="https://www.targetvalidation.org/variants">variants</a>, normalised p-value and normalised sample size); European Variation Archive (functional consequence score of variants e.g. germline variants that cause transcrip ablation will have a score of 1, whereas variants that are intronic will have a score of 0.5); UniProt (curator inference score based on how strong the evidence for the gene's involvement in the disease is. If the evidence is strong, the score will be 1. For evidence deemed not to be strong by the curator, the score will be 0.5); Gene2Phenotype (variants are inferred by curators and will have a score of 1, the highest functional consequence score); Genomics England PanelAPP (gene-disease associations are curated and crowdsourced by experts and will have the highest score of 1)</td>
                    </tr>

                    <tr>
                        <td>Somatic mutations</td>
                        <td>Cancer Gene Census (functional consequence score of variants); European Variation Archive (functional consequence score of variants); IntOgen (binned score based on tumour type categories. If the gene has several signals of positive selections in the tumour, the score will be 0.25. If the gene is already described as a cancer gene and exhibits a signal of positive selection in a tumor type, the score will be 0.5. If in addition to a signal of positive selection, the gene is functionally connected to other genes in the same tumor type, the score will be 0.75)</td>
                    </tr>

                    <tr>
                        <td>Drugs</td>
                        <td>ChEMBL  (Clinical trials phase binned score. Scores will be 0.09 for phase 0, 0.1 for phase I, 0.2 for Phase II, 0.7 for Phase III, and 1 for Phase IV drugs)</td>
                    </tr>

                    <tr>
                        <td>Affected pathways</td>
                        <td>Reactome (functional consequence of 1 for a pathaway inferred by a curator)</td>
                    </tr>

                    <tr>
                        <td>RNA Expression</td>
                        <td>Expression Atlas score (normalised p-value, normalised expression fold change and normalised percentile rank)</td>
                    </tr>

                    <tr>
                        <td>Text mining</td>
                        <td>Europe PMC (weighting document sections, sentence locations and title for full text articles and abstracts (<a target=_blank href="https://opentargets.org/sites/default/files/documents/literature_evidence_in_cttv_-_a_target_validation_platform.pdf">Kafkas et al 2016</a>)</td>
                    </tr>

                    <tr>
                        <td>Animal models</td>
                        <td><a href="http://database.oxfordjournals.org/content/2013/bat025">Phenodigm</a> (<a href="http://database.oxfordjournals.org/content/2013/bat025">similarity score between a mouse model and a human disease</a><a href="http://database.oxfordjournals.org/content/2013/bat025.full#sec-3"> described by Smedley et al 2013)</a>)</td>
                    </tr>

                </tbody>
            </table>

            <p>Once we have the scores for each evidence, we calculate an overall score for a data type (e.g. Genetic associations). In this step, we take into account that although multiple occurrences of evidence can suggest a strong association, the inclusion of further new evidence should not have a great impact on the overall score. For this reason, we calculate the sum of the <a href="https://en.wikipedia.org/wiki/Harmonic_progression_(mathematics)">harmonic progression</a> of each score and adjust the contribution of each of them using a heuristic weighting. Throughout this process, the value of the score is always capped at 1, the most confident association.</p>

            <p>We have implemented this framework for the current version of the Platform (please refer to <a href="http://nar.oxfordjournals.org/content/early/2016/11/29/nar.gkw1055.full">&quot;Open Targets: a platform for therapeutic target identification and validation&quot;</a> for more details. We will continue to explore and work on alternative statistical models to keep providing robust scoring systems for target-disease associations.</p>

        </div>

    </div>

